scripts/inner/kubestellar

```diff
@@ -115,6 +115,7 @@ function ensure_wmws() {
             fi
             wmw=${i#"root:"}
             echo "Ensuring WMW: $wmw"
+            # TODO: we shouldn't hard code with-kube
```

Comment:
Maybe also add a TODO of what we need to do.

Reply:
done.  I already have the fix ready to PR.  Just waiting for this to get merged  :)